lib/check-base-branch.js

```diff
@@ -20794,49 +20794,49 @@ var require_github = __commonJS((exports2) => {
 // src/check-base-branch.ts
 var core = __toModule(require_core());
 var github = __toModule(require_github());
-var getPRNumber = () => {
-  const pr = github.context.payload.pull_request;
-  if (pr) {
-    return pr.number;
-  }
+var getPR = () => {
+  return github.context.payload.pull_request;
 };
 var getProtectedBranches = () => {
   const branches = core.getInput("protected-branches", {required: true});
   return branches.split(",").map((b) => b.trim());
 };
 function run() {
   return __async(this, null, function* () {
-    const prNumber = getPRNumber();
-    if (!prNumber) {
-      core.debug("Could not get pull request number from context. Skipping.");
+    const pr = getPR();
+    if (!pr) {
+      core.debug("Could not get pull request from context. Skipping.");
       return;
     }
     try {
-      const token = core.getInput("repo-token", {required: true});
       const protectedBranches = getProtectedBranches();
       const updateBranch = core.getInput("update-branch");
-      const defaultBranch = core.getInput("default-branch", {required: updateBranch !== "true"});
-      const oktokit = new github.GitHub(token);
-      core.debug(`Checking base branch for PR #${prNumber}`);
-      const payload = {
-        owner: github.context.repo.owner,
-        repo: github.context.repo.repo,
-        pull_number: prNumber
-      };
-      const pr = yield oktokit.pulls.get(__assign({}, payload));
-      if (protectedBranches.includes(pr.data.base.ref)) {
+      const defaultBranch = core.getInput("default-branch", {
+        required: updateBranch !== "true"
```

Comment:
Here & below: should these ``updateBranch === "true"``?

Reply:
You are right. This was always `!==` and I guess I never caught it!